bedrock/settings/static_media.py

```diff
@@ -1301,6 +1316,29 @@
         ),
         'output_filename': 'js/firefox_firstrun-bundle.js',
     },
+    'firefox_firstrun_quantum': {
+        'source_filenames': (
+            'js/base/uitour-lib.js',
+            'js/base/mozilla-fxa-iframe.js',
+            'js/firefox/firstrun/firstrun_quantum.js',
+        ),
+        'output_filename': 'js/firefox_firstrun_quantum-bundle.js',
+    },
+    'experiment-firstrun-membership': {
```

Comment:
This and the `'firefox_firstrun_membership'` block below were also removed from master and shouldn't be readded here.